packages/shared/src/getTypegenOutput.ts

```diff
@@ -202,20 +202,13 @@ const collectInternalEvents = (lineArrays: { events: string[] }[][]) => {
 const displayEventsCausing = (lines: { name: string; events: string[] }[]) => {
   return lines
     .map((line) => {
-      return `${JSON.stringify(line.name)}: ${
-        unique(
-          line.events.map((event) => {
-            return event;
-          })
-        )
-          .map((event) => JSON.stringify(event))
-          .join(" | ") ||
-        /**
-         * If no transitions go to this guard/service/action, it's guaranteed
-         * to be caused by xstate.init.
-         */
-        "'xstate.init'"
-      };`;
+      return `${JSON.stringify(line.name)}: ${unique(
+        line.events.map((event) => {
+          return event;
+        })
+      )
+        .map((event) => JSON.stringify(event))
```

Comment:
we might consider sorting those here as otherwise the generated file gets sensitive to our internal logic - this might be observed in one of the updated snapshots in this PR since the events got merely reordered there